pyccel/stdlib/cuda_ndarrays/cuda_ndarrays.cu

```diff
@@ -0,0 +1,170 @@
+#include "cuda_ndarrays.h"
+
+__global__
+void cuda_array_arange_int8(t_ndarray arr, int start)
+{
+    int index = blockIdx.x * blockDim.x + threadIdx.x;
+    int stride = gridDim.x * blockDim.x;
+
+    for(int i = index ; i < arr.length; i+=stride)
+        arr.nd_int8[i] = (i + start);
+}
+__global__
+void cuda_array_arange_int32(t_ndarray arr, int start)
```

Comment:
I find it odd that you have `int8` and `int32` but not `int16`. Is this intentional?